sfcasts/dashboard.md

```diff
@@ -1,95 +1,61 @@
-This leverages the font awesome library. So `fa-dashboard] corresponds to that little
-icon right there. All right. So we can definitely do more with our dashboard, but
-that's enough for now because what we're really here for are the CRUD controllers.
-These are these sections of our site that it's going to allow us to create, read,
-update, and delete all of our entities. Let's get those going next.
+As a reminder, `symfony console` is exactly the same as running `php bin/console`. The only difference is that running `symfony console` allows the Docker environment variables to be injected into this command. It generally makes no difference unless you're running a command that requires database access. So in this case, `php bin/console` would work just fine.
+
+I'll stick with `symfony console` throughout this tutorial. So say:
+
+```terminal
+symfony console make:admin:dashboard
+```
+
+We'll call it "DashboardController."
+
+Sounds good! To generate it, say `src/Controller/admin` and... done! This creates one new file for us: `src/Controller/Admin/DashboardController.php`.
+
+Let's go check it out!
+
+Ooh... Let me see... There's not a lot here, but one thing you might notice is that it has a route for `/admin`. So now if I go over here to `/admin`... we do hit the admin dashboard!
+
+Okay, so there's a few things to notice here. First is that we do have a `/admin` route. This isn't fancy, and there's really nothing "EasyAdmin" about it. This is just how we create routes in Symfony. This is a PHP 8 attribute route, which you may or may not be familiar with. I've typically used annotations until now, but now that I'm using PHP 8, I'll be using attributes instead of annotations. Don't worry, though! They work exactly the same. If you're still using PHP 7, then you can use annotations just fine.
+
+The `DashboardController` is just a normal controller, but it extends `AbstractDashboardController`. I'll hold "Cmd" and open that. 
+
+This implements `DashboardControllerInterface`. So this is a controller, but by implementing the interface, it gives it special superpowers. There are a number of methods inside of here that we can override to configure what our dashboard looks like. We'll be doing that little by little throughout this tutorial. 
+
+Because this is just a normal route in controller, there's really nothing special about it. It's not automatically secured and there's no security being applied to this. I mean, check it out: I'm not even logged in right now, and I am successfully on my admin dashboard.
+
+So let's secure it! I'll also do this with annotations. I already have SensioFrameworkExtraBundle installed. So I can say `#[IsGranted()]` and hit "tab" to auto complete that. Now we'll look for `ROLE_ADMIN`. That's kind of my base admin role, so all admin users will have that role. Now when I refresh... Beautiful! We bounced back over to the login page!
```

Comment:
"so all admin users will have that."

This avoids 2x role" in the same sentence, and the 2nd isn't needed for clarity.